runtime/doc/eval.txt

```diff
@@ -7825,6 +7826,11 @@ reg_executing()						*reg_executing()*
 		Returns an empty string when no register is being executed.
 		See |@|.
 
+reg_recorded()						*reg_recorded()*
+		Returns the single letter name of the last recorded register.
+		Returns an empty string string when nothing was recorded yet.
```

Comment:
string is repeated twice in this line (and `reg_recording` below). Is this a typo?

Reply:
You can make a PR to fix it.

Reply:
I'll fix it.